ultrack/core/export/exporter.py

```diff
@@ -19,11 +21,12 @@ def export_tracks_by_extension(
     """
     Export tracks to a file given the file extension.
 
-    Supported file extensions are .xml, .csv, .zarr, .dot, and .json.
+    Supported file extensions are .xml, .csv, .zarr, .parquet, .dot, .json, and .geff.zarr.
```

Comment:
```suggestion
    Supported file extensions are .xml, .csv, .zarr, .parquet, .dot, .json, and .geff
```
same as above, it depends on what we decide